pkgs/development/libraries/unixODBCDrivers/default.nix

```diff
@@ -181,6 +181,104 @@
     };
   };
 
+  msodbcsql18 = stdenv.mkDerivation rec {
+      pname = "msodbcsql${versionMajor}";
+      version = "${versionMajor}.${versionMinor}.${versionAdditional}${versionSuffix}";
+
+      versionMajor = "18";
+      versionMinor = "1";
+      versionAdditional = "1.1";
+      versionSuffix = lib.strings.optionalString stdenv.targetPlatform.isLinux "-1";
+
+    src =
```

Comment:
I like this.

Happy to use `hash = "sha256:..."`; I wasn't aware of it. I kept the base-64 encoding because that's what `nix-prefetch-url` gives me.